Wrappers.h

```diff
@@ -0,0 +1,40 @@
+#include "Knife.h"
+#include "Light.h"
+#include "SwissArmyKnife.h"
+#include "KnifeShop.h"
+#include "KillRoom.h"
```

Comment:
You aren't using any of these in this header file.  you're using them in the cpp file.
move the includes.  add forward declarations.